src/OMSimulatorLib/Component.h

```diff
@@ -69,6 +70,7 @@ namespace oms
     const std::string& getPath() const {return path;}
     oms_component_enu_t getType() const {return type;}
     virtual const FMUInfo* getFMUInfo() const {return NULL;}
+    virtual const ExternalModelInfo* getExternalModelInfo() const {return NULL;}
```

Comment:
Not sure if we should really have `getFMUInfo` and  `getExternalModelInfo` in class Component or not... 🤔

Reply:
I guess there is no need, since each of these functions are only used in a single derived class. I can fix it when I'm back from my summer vacation.

Reply:
I removed `getExternalModelInfo` from `Component` class. I'm not sure how to solve it for `getFMUInfo`, since it is used in two subclasses. It doesn't really belong to this PR anyway, so I leave it for now.